packages/rn-tester-e2e/package.json

```diff
@@ -15,8 +15,6 @@
   },
   "devDependencies": {
     "appium": "^2.0.0",
```

Comment:
We do a separate global Appium install it looks like? We should either remove from devDependencies, or use locally installed version when we are running the test.

Reply:
+1

Reply:
I removed from `package.json`, let's see how it goes. cc. @kelset you should know about this change, would be great if you confirm is this change safe 🙏

Reply:
ah sorry I left a comment on the side. We shouldn't remove the ones in devDiv, it might be enough to remove the install step from CI

Reply:
I'm pretty sure that it won't work. We're starting a server with `appium` command and globally installed won't contain drivers so it will fail.

Reply:
It won't work because we're starting appium server with globally installed command, so that it will look for drivers installed and it will fail as there will be no drivers installed. I tested that locally and it is failing.

Reply:
Why use a global command vs running the locally installed version via npx? That way, all transitive dependencies are versioned (at the cost of bloating this repos lockfile more). 

https://appium.io/docs/en/2.0/guides/managing-exts/